opencensus/src/test/scala/zio/telemetry/opencensus/OpenCensusTracingTest.scala

```diff
@@ -0,0 +1,65 @@
+package zio.telemetry.opencensus
+
+import io.opencensus.trace.`export`.SpanData
+import io.opencensus.trace.samplers.Samplers
+import io.opencensus.trace.{ SpanId, Tracer, Tracing => OTracing }
+import zio._
+import zio.test.Assertion._
+import zio.test._
+
+object OpenCensusTracingTest extends ZIOSpecDefault {
```

Comment:
Could you please rename it to `TracingTest` in order to keep the same naming style across all modules?

Reply:
done